### PR TITLE
[JN-1232] AT webpage changes

### PR DESCRIPTION
**Pull request by connorlbark**
#### DESCRIPTION (include screenshots, and mobile screenshots for participant UX)

Adds the ability to make the image the background in a hero with image template:
Screenshot 2024-07-29 at 12 21 59 PM

As with the AT website, on mobile the background image will disappear.

image

Also, the A-T website is very text heavy. The text was much too large, so I added the ability to change the blurb text size:

Screenshot 2024-07-29 at 12 22 29 PM

Admin side:
image

I'm open to changing `blurbSize` to not be the bootstrap class names... but I figured it was intuitive enough once you clicked on it and observed the change.

#### TO TEST:  *(simple manual steps for confirming core behavior -- used for pre-release checks)*

Download and populate [this portal](https://drive.google.com/file/d/1dge2FSXeBUznffN9MNzkK2YY7QqzFnU8/view?usp=drive_link)

Go to [https://sandbox.atcp.localhost:3001](https://sandbox.atcp.localhost:3001) and check out all of the navbar items. Compare them to https://atcp.dev.datadonationplatform.org/ (note that we don't have the DAR page, I figured I would put that off for now).

Go to the admin side and ensure that `blurbSize` works as expected, and that `imageAsBackground` makes sense.

### CODE REVIEW

File: ui-core/src/participant/landing/sections/HeroWithImageTemplate.tsx [hunk 1]
```diff
@@ -121,14 +168,14 @@ function HeroWithImageTemplate(props: HeroWithImageTemplateProps) {
...
-            <ConfiguredMedia media={image} className="img-fluid"/>
+            {!imageAsBackground && <ConfiguredMedia media={image} className="img-fluid"/>}
           </div>
         )}
         <div
           className={classNames(
             'col-12', `col-lg-${12 - imageCols}`,
             'py-3 p-sm-3 p-lg-5',
-            'd-flex flex-column flex-grow-1 justify-content-around'
+            'd-flex flex-column flex-grow-1 justify-content-center'
```
Comment (connorlbark) [inline ui-core/src/participant/landing/sections/HeroWithImageTemplate.tsx:178]: This should be the only change that actually effects how things look for production user. I think `justify-content-center` looks nicer in general (and especially for the AT landing page), and it should be a minor change of just a few pixels of spacing for the most part, but we should definitely go through all of the portals in detail to make sure the changes don't look off.

Comment (connorlbark) [inline ui-core/src/participant/landing/sections/HeroWithImageTemplate.tsx:178]: same as [matt's pr](https://github.com/broadinstitute/juniper/pull/1027); no other changes should have any visible impact to existing site contents

File: ui-admin/src/portal/siteContent/designer/editors/StyleEditor.tsx [hunk 2]
```diff
@@ -62,6 +85,25 @@ export const StyleEditor = ({ section, updateSection }: {
                   updateSection({ ...section, sectionConfig: JSON.stringify({ ...config, blurbAlign: opt?.value }) })
                 }}/>
             </div>}
+        {Object.hasOwnProperty.call(config, 'blurb') && <div>
+          <label className="form-label fw-semibold">Blurb Text Size</label>
```
Comment (devonbush) [inline ui-admin/src/portal/siteContent/designer/editors/StyleEditor.tsx]: use the `useReactSingleSelect` helper, especially using the `selectInputId` returned -- that will let you make the label accessible

File: ui-core/src/participant/landing/sections/HeroWithImageTemplate.tsx [hunk 3]
```diff
@@ -111,8 +147,19 @@ function HeroWithImageTemplate(props: HeroWithImageTemplateProps) {
           'row',
           'col-12',
           fullWidth ? 'mx-0' : 'col-sm-10 mx-auto',
-          isLeftImage ? 'flex-row' : 'flex-row-reverse'
+          isLeftImage ? 'flex-row' : 'flex-row-reverse',
+          imageAsBackground ? 'responsive-bg-image' : ''
         )}
+        style={imageAsBackground ? {
```
Comment (devonbush) [inline ui-core/src/participant/landing/sections/HeroWithImageTemplate.tsx:153]: this code section feels like too many lines for a ternary, but I guess it's nice getting to define most of the style props here

**Review by devonbush — APPROVED** [review]
ncie changes -- thanks for adding editor support for it too.  some code cleanup -- can you make sure you add "do visual comparison of OurHealth prod/demo" to the team testing doc todo for tomorrw

File: ui-core/src/participant/landing/sections/HeroWithImageTemplate.tsx (hunk 3 above)
Comment (connorlbark) [reply ui-core/src/participant/landing/sections/HeroWithImageTemplate.tsx:153]: I don't feel strongly, but I agree it's nice to see it inline. Open to alternatives.

File: ui-core/src/participant/landing/sections/HeroWithImageTemplate.scss [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+@media (max-width: 991.98px) {
```
Comment (MatthewBemis) [inline ui-core/src/participant/landing/sections/HeroWithImageTemplate.scss:2]: this max-width looks oddly specific. 😄 where did it come from?

**Review by MatthewBemis — APPROVED** [review]

**Pull request merged**